docs/reference/config-params.md

```diff
@@ -805,124 +932,157 @@ Again, it is ideal for exposing a service or app to the internet under a single
 
 Practically, in non development environments, a LoadBalancer will be used to route traffic to an Ingress to expose multiple services under the same IP address and keep your costs down.
 
+## x-k8s.service.nodeport
 
-## kev.service.nodeport.port
-
-Defines type Node Port value for Kubernetes service of `NodePort` type. See official K8s [documentation](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport).
+Defines the Node Port value for a Kubernetes service of type `NodePort`. See official K8s [documentation](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport).
 NOTE: `nodeport` attributes will be ignored for any other service type!
 Kev will attempt to extract that information from the compose configuration.
 
 ### Default: `nil` - no nodeport defined by default!
 
 ### Possible options: Arbitrary integer. Example `10222`.
 
-> kev.service.nodeport.port:
+> x-k8s.service.nodeport:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.service.nodeport.port: 5555
+    x-k8s:
+      service:
+        type: nodeport
+        nodeport: 5555
 ...
 ```
 
-## kev.service.expose
+## x-k8s.service.expose
 
-Defines whether to expose the service to external world. This detail can't be easily derived from the compose file and so in order to expose a service to external world user must explicitly instruct Kev to do so. By default all component services aren't exposed i.e. have no ingress attached to them.
+Defines how to expose the service externally. This detail can't be easily derived from the compose file and so in order to expose a service the user must explicitly instruct Kev to do so. By default, all component services aren't exposed i.e. have no ingress attached to them.
 
-### Default: `""` - No ingress will be created!
+### x-k8s.service.expose.domain
 
-### Possible options:
-* `"true"` - ingress will be created with Kubernetes cluster defaults
```

Comment:
I think it's useful to leave that info in the docs.

Reply:
As discussed, `true` will be treated as a new feature in upcoming ticket.

Reply:
ok, I guess the docs change related to that will be also in the upcoming PR?

Reply:
Will fix now as its been finalised.